types/index.d.ts

```diff
@@ -1538,6 +1538,12 @@ declare namespace RamdaAdjunct {
         toUinteger32(val: number): number;
         toUint32(val: number): number; // alias
 
+        /**
+         * Converts value to a number.
+         * It returns `NaN` whenever the global `Number` function would return `NaN`.
```

Comment:
No need for this additional sentence. This sentence only reveal additional property of our `toNumber` function. More import is IMHO the fact that the function is safe and never throw errors, apart from `Number` function from javascript.
```suggestion
```